cfgov/apache/conf.d/redirects.conf

```
@@ -982,19 +930,18 @@ RedirectMatch permanent (?i)^/es/guia-asesor-financiero https://pueblo.gpo.gov/C
 ## This one specific page got moved to another specific page
 RedirectMatch permanent (?i)^/older-americans/protecting-whats-yours([/]?)$ /practitioner-resources/resources-for-older-adults/protecting-whats-yours/
 
-## Send everything else beginning with /older-americans to the new main Resoures for Older Adults page
+## Send everything else beginning with /older-americans to the new main Resources for Older Adults page
 RedirectMatch permanent (?i)^/older-americans/ /practitioner-resources/resources-for-older-adults/
 
 ## MSEM moved under Resources for Older Adults
 RedirectMatch permanent (?i)^/managing-someone-elses-money([/]?)$ /consumer-tools/managing-someone-elses-money/
 
-
 # Redirect old Tax Preparers URL to its new Wagtail home
 RedirectMatch permanent (?i)^/tax-preparer-resources([/]?)$ /practitioner-resources/resources-for-tax-preparers/
 
 # Redirect FY13-17 Strategic Plan page to PDF to move out of WordPress
 # See discussion here: https://GHE/CFGOV/platform/issues/710
-Redirect /strategic-plan/ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
+RedirectMatch permanent (?i)^/strategic-plan/$ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
```

Let's not make this permanent. This URL may be reused to point to future strategic plans.

```suggestion
RedirectMatch (?i)^/strategic-plan/$ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
```